### ISSUE TITLE
Irony -DotNetcore- Missing features

### ISSUE THREAD

**Issue by kalaiselviA** [daxnet/irony#2]
I could see these features DynamicCallDispatcher,EvaluationContext,datastack,StackFrame,Valuetable   are currently not available in Irony DotNetcore source code.Could you please let me know why it has removed and do we have any alternative for this.

**Comment by kalaiselviA** [comment]
Do anybody have idea about this issue in .netcore

**Comment by daxnet** [comment]
@kalaiselviA Good catch! To be honest for now this Irony repo is a simple migration from its original Codeplex repo, and I didn't get a chance to look into the details. I will try to do that later.

**Comment by kalaiselviA** [comment]
Yeah Please let me know , if you have done it
